packages/components/rate/src/rate.vue

```diff
@@ -242,10 +241,10 @@ export default defineComponent({
       }
       if (props.allowHalf) {
         let target = event.target as HTMLElement
-        if (hasClass(target, 'el-rate__item')) {
-          target = target.querySelector('.el-rate__icon')!
+        if (hasClass(target, ns.e('item'))) {
```

Comment:
Can we cache them via computed?

Reply:
This feature is not the main goal of this PR.

Reply:
Would you mind add a `TODO` comment for further steps? In case we forgot, don't forget to include this conversation.

Reply:
Updated.